### ISSUE TITLE
Global Search

### ISSUE THREAD

**Issue by STRATZ-Ken** [spheenik/clarity-analyzer#10]
Currently we have a feature to search within the total list of Entitys for a specific Entity.  Would love a feature that could global search all Entity for a specific word.  For example, I am trying to find the SeriesId of the match, would be great to search `series` and see any Entity that has that word within it.

**Comment by spheenik** [comment]
I agree this is a needed feature. Will see that I come around to implementing it.

**Comment by spheenik** [comment]
Would still be a nice feature though, to put a filter box for the properties as well. Sure implement it, and send a PR ;-)
